packages/docs/src/shared-components/appshell.ts

```diff
@@ -0,0 +1,340 @@
+      }
+    }
+  </style>
+  <footer class="synergy-footer-demo">
+    <div class="footer-content" aria-label="Footer navigation">
+      <nav class="link-wrapper">
+        <a class="syn-link syn-link--medium syn-link--quiet" href="${getTranslation('footer.linksHref.imprint')}">${getTranslation('footer.linksName.imprint')}</a>
+        <a class="syn-link syn-link--medium syn-link--quiet" href="${getTranslation('footer.linksHref.termsConditions')}">${getTranslation('footer.linksName.termsConditions')}</a>
+        <a class="syn-link syn-link--medium syn-link--quiet" href="${getTranslation('footer.linksHref.termsOfUse')}">${getTranslation('footer.linksName.termsOfUse')}</a>
+        <a class="syn-link syn-link--medium syn-link--quiet" href="${getTranslation('footer.linksHref.privacyPolicy')}">${getTranslation('footer.linksName.privacyPolicy')}</a>
+      </nav>
+      <p class="copyright">&copy; 2024 SICK AG</p>
+    </div>
+  </footer>
+  <!-- /footer -->
+`;
+
+/**
+ * Create shared styles for all stories
+ */
+export const createSharedStyles = () => html`
+  <style>
+    body {
+      margin: 0 !important;
+      padding: 0 !important;
+    }
+
+    #storybook-root,
+    #root-inner {
+      height: 100%;
+    }
+
+    .synergy-demo-application {
+      container-type: inline-size;
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      min-height: 550px;
+    }
+
+    .synergy-demo-content {
+      background: var(--syn-color-neutral-100);
+      display: flex;
+      flex: 1 0 auto;
+      flex-direction: column;
+      height: 1%;
+      position: relative;
+    }
+
+    .synergy-demo-content-inner {
+      container-type: inline-size;
+      display: flex;
+      flex: 1 1 auto;
+      flex-direction: column;
+    }
+
+    .synergy-demo-main {
+      align-items: center;
+      border: 1px dashed #9747FF;
+      border-radius: var(--syn-border-radius-medium);
+      color: #9747FF;
+      display: flex;
+      flex: 1 0 auto;
+      flex-direction: column;
+      font: var(--syn-body-small-bold);
+      justify-content: center;
+      margin: var(--syn-spacing-2x-large) var(--syn-spacing-2x-large) 0;
+      padding: 0 var(--syn-spacing-small);
+    }
+
+    @container (max-width: 768px) {
+      .synergy-demo-main {
+        margin: var(--syn-spacing-medium) var(--syn-spacing-medium) 0;
+      }
+    }
+
+    .synergy-demo-main h1 {
+      color: var(--syn-typography-color-text);
+      font: var(--syn-heading-large);
+      margin: 0;
+      text-align: center;
+    }
+
+    .synergy-demo-footer > nav {
+      align-items: center;
+      display: flex;
+      font: var(--syn-body-x-small-regular);
+      gap: var(--syn-spacing-large);
+      height: var(--syn-spacing-3x-large);
+      justify-content: center;
+    }
+
+    .synergy-demo-footer :is(a, strong) {
+      color: var(--syn-input-help-text-color);
+    }
+
+    .synergy-demo-footer a:hover {
+      color: var(--syn-color-primary-600);
```

Comment:
As this may be removed by a reset somewhere else: Do we want to have a `text-decoration: underline;` here, too? For accessability reasons, I think it would be mandatory to not only work with color during hover.

Reply:
just saw, that this is not needed anyways, because I created the shared footer style and this is therefore no longer used

Reply:
As this is using our `syn-link`, it may not be needed at all :)